openbb_platform/CONTRIBUTING.md

```diff
@@ -287,145 +446,68 @@ class <ProviderName>StockEODFetcher(
     @staticmethod
     def transform_data(
         data: dict,
-    ) -> List[<ProviderName>StockEODData]:
+    ) -> List[<ProviderName>StockHistoricalData]:
         """Transform the data to the standard format."""
 
-        return [<ProviderName>StockEODData.model_validate(d) for d in data]
-
+        return [<ProviderName>StockHistoricalData.model_validate(d) for d in data]
 ```
 
 > Make sure that you're following the TET pattern when building a `Fetcher` - **Transform, Extract, Transform**.
 
-### Make the new provider visible to the Platform
+### Make the provider visible
 
-In order to make the new provider visible to the Platform, you'll need to add it to the `__init__.py` file of the `providers/<provider_name>/openbb_<provider_name>/` folder.
+In order to make the new provider visible to the OpenBB Platform, you'll need to add it to the `__init__.py` file of the `providers/<provider_name>/openbb_<provider_name>/` folder.
 
 ```python
-
 """<Provider Name> Provider module."""
 from openbb_provider.abstract.provider import Provider
 
-from openbb_<provider_name>.models.stock_eod import <ProviderName>StockEODFetcher
+from openbb_<provider_name>.models.stock_eod import <ProviderName>StockHistoricalFetcher
 
 <provider_name>_provider = Provider(
     name="<provider_name>",
     website="<URL to the provider website>",
     description="Provider description goes here",
     required_credentials=["api_key"],
-    `Fetcher`_dict={
-        "StockEOD": <ProviderName>StockEODFetcher,
+    fetcher_dict={
+        "StockHistorical": <ProviderName>StockHistoricalFetcher,
     },
 )
-
 ```
 
-If the provider does not require any credentials, you can simply omit it. On the other hand if it requires more than 2 items to authenticate, you can simply add a list of all the required items to the `required_credentials` list.
+If the provider does not require any credentials, you can remove that parameter. On the other hand, if it requires more than 2 items to authenticate, you can add a list of all the required items to the `required_credentials` list.
```

Comment:
Previously, this parameter needed to exist even if the value was empty.  Has this changed?

Reply:
Yes, it is an optional parameter.